kk/models/images.py

```diff
@@ -15,16 +12,16 @@ def get_images_dir():
 file_storage = FileSystemStorage(location=get_images_dir())
 
 
-class CommonImage(ModifiableModel):
+class Image(ModifiableModel):
```

Comment:
Having M2M relation per each model (hearing, introduction, scenario, more) seems to be more readable than image per model with conflicting relations (through 'related_name').